onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -135,15 +135,6 @@ def run_onnxruntime(
             )
             return results
 
-    if provider == "migraphx":
-        optimizer_info = OptimizerInfo.NOOPT
-        warm_up_repeat = 5
-        if "MIGraphXExecutionProvider" not in onnxruntime.get_available_providers():
```

Comment:
This warning might still be helpful?

Reply:
Its just reverting things from the previous state